### DISCUSSION TITLE
Latest commit seems to break which-key checkhealth after telescope is loaded.

### DISCUSSION THREAD

**Discussion by MrRothstein** [LazyVim/LazyVim#1646] [General]
Running checkhealth on startup works as expected, but running it after using Telescope results in:

ERROR Failed to run healthcheck for "which-key" plugin. Exception:
  function health#check, line 25
  Vim(eval):E5108: Error executing lua ...al/share/nvim/lazy/which-key.nvim/lua/which-key/keys.lua:405: Invalid buffer id: 1
  stack traceback:
  [C]: in function 'nvim_buf_get_keymap'
  ...al/share/nvim/lazy/which-key.nvim/lua/which-key/keys.lua:405: in function 'update_keymaps'
  .../share/nvim/lazy/which-key.nvim/lua/which-key/health.lua:15: in function 'check'
  [string "luaeval()"]:1: in main chunk

Sorry if this is the wrong place to report it.

**Comment by MrRothstein** [comment]
Nevermind, it's also happening after restore... Not sure what is causing it and it doesn't seem to happen every time.

**Comment by ViniciusLagoGehrke** [comment]
I'm getting the same issue and cannot run the healthcheck properly. The buffer id is not always the same.
